<launch>
    <node pkg="roboclaw_driver" type="roboclaw_driver_node" name="roboclaw_driver" required="true" output="screen">
		<param name="acceleration_limit" value="0.5"/>
		<param name="encoder_res" value="32000"/>
	</node>
</launch>